app/resonant-laboratory/ansible/xenial/girder.service.template

```diff
@@ -0,0 +1,7 @@
+[Unit]
+Description=Girder
+After=network.target
+
+[Service]
```

Comment:
At a minimum I would put a User/Group here so Girder doesn't run as root. Example [here](https://github.com/girder/girder/blob/master/devops/ansible/roles/girder/templates/daemon/girder.service.j2#L6-L7). Same goes for the upstart config.